### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

#84 Update api version support to 67 
#83 Update Readme based on the new template 
#80 Add new APIs 

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

**Comment by Aleffio** [comment]
please also update
https://github.com/Adyen/adyen-ruby-api-library/blob/5bae341b69e7600b375f8d470e27b6660635c1ea/lib/adyen/version.rb#L3

File: README.md [hunk 1]
```diff
@@ -1,55 +1,71 @@
...
+~~~~
+~~~~ruby
 adyen = Adyen::Client.new
 
-# for API-key based implementations
 adyen.api_key = 'AF5XXXXXXXXXXXXXXXXXXXX'
 
-# for basic-auth based implementations
 adyen.ws_user = 'ws@Company.Adyen'
 adyen.ws_password = 'super_secure_password123'
```
[the remark this answers is not in the document]
Comment (AlexandrosMor) [reply README.md]: removed

File: README.md [hunk 2]
```diff
@@ -66,114 +82,37 @@ response = adyen.checkout.payments({
   :returnUrl => "https://your-company.com/checkout/",
```
[the remark this answers is not in the document]
Comment (AlexandrosMor) [reply README.md:80]: ok done

**Review by Aleffio — APPROVED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1 @@
-* @crrood @hbkwong @KadoBOT @Aleffio @rikterbeek
\ No newline at end of file
+* @crrood @Aleffio @wboereboom @cyattilakiss
```
Comment (wboereboom) [inline .github/CODEOWNERS:1]: Please also add @zaiddreakh

**Review by wboereboom — APPROVED** [review]
🎉

**Pull request merged**